_posts/2023-08-31-Minutes-of-SCM-28.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  "Minutes of Steering Committee Meeting No.28"
+from:   2023-08-31 00:00:00 +1000    
+to:     2023-08-31 00:00:00 +1000    
+tags: ["steering Committee", "2023"]
+published: true                     
+---
+
+## Attending:
+- MS
+- RM
+- JB
+- JS
+
+
+## Apologies
+- NL
+
+
+## Follow up on action items from last meeting:
+- M to create two separate documents in order to apply for fiscal sponsorship to Open Life Sciences (OLS)
+- i) Succession plan: this document will ensure the committee's operationalibilty in case of SC members stepping down.
+- ii) Dispute resolution and linked from the governance document. Find then [Dispute resolution document on the website](https://hackmd.io/@manodeep/rse-aunz-dispute-resolution/edit) here.
+
+#### Discussion on the dispute resolution process
+- An agreement has been made on steps  1-7 of the dispute resolution process. 
+- action item: go back to OLS to clarify what the final tie breaker should look like (a person, random choice etc).
+
+#### Discussion on succession plan
+* An agreement of succession has been made on most scenarios of single or multiple SC members stepping down. Details can be found in the Succession plan document.
+
+
+# Updates on running items:
+## Update on NZRSE Conference 2023
+- talks have been approved and decided on
+- See here for the conference [website](https://www.rseconference.nz/)
+
+
+## RSEAA2023 
+- We are getting ready for the event. 
+- Had to use a personal credit card for Zoom Events.
+
+## Face to Face meetings 
+- This is ongoing. We want to email this as part of the regular monthly email.
+
+
+## Senior Advisory Board 
+
+- Going to be treated as 2024 thing. I might chat unofficially with senior panelists from RSEAA2023 if I get the chance though. 
+
+
+
+## RSE UK submission 
+
+- Johanna is going and going to give a 2 min talk on RSEAUNZ in the leaders meeting
+
+## Getting new SC members 
+- we need new steering committee members 
+- we need an advisory panel, and we regular members
```

Comment:
This sentence needs to be fixed

Reply:
we need an advisory panel, and regular members

woudl that be ok?

Reply:
I pushed two commits to fix this.